chart/README.md

```diff
@@ -20,17 +20,18 @@ recommend deploying your main NATS server with another chart.
 
 ## Running the chart
 
-The chart is available in a hosted Helm repository. To add the repository, run the following:
+You can install the chart via [the `wasmcloud-chart` package on ArtifactHub][artifacthub-wasmcloud]:
 
 ```console
-$ helm repo add wasmcloud https://wasmcloud.github.io/wasmcloud-otp/
+$ helm install wasmcloud oci://ghcr.io/wasmcloud/wasmcloud-chart --version 0.7.0
 ```
 
-Then you can install the chart:
-
-```console
-$ helm install <RELEASE_NAME> wasmcloud/wasmcloud-host
-```
+> [!NOTE]
+>
+> You can replace `wasmcloud` with whatever you'd like to call your instantiation
+> of the helm chart.
+>
+> Version `0.7.0` of the helm chart (released Nov 9th, 2023) deploys version [`v0.80.0`][wasmcloud-v0.80.0] of the host by default.
```

Comment:
same as above

Reply:
Would it be worth mentioning that the default version (as provided by `appVersion`) can be overridden by setting `wasmcloud.image.tag` value in values.yaml?

Reply:
Done! mentioned the values override